test-suite/unit-tests/src/utest.ml

```diff
@@ -43,6 +43,7 @@ let run_one logit test =
   process_results results
 
 let open_log_out_ch ml_fn =
+  let ml_fn = Filename.basename ml_fn in
```

Comment:
This breaks the makefile rules to run the tests, no?

Reply:
Yes, we need to tweak that part. The current setup for the ounit testing runner seems could use quite a lot of work.